programs/autocrat_v0/src/lib.rs

```diff
@@ -381,18 +388,44 @@ pub mod autocrat_v0 {
         Ok(())
     }
 
-    pub fn set_last_proposal_slot(ctx: Context<Auth>, last_proposal_slot: u64) -> Result<()> {
+    pub fn set_base_burn_lamports(ctx: Context<Auth>, base_burn_lamports: u64) -> Result<()> {
         let dao = &mut ctx.accounts.dao;
 
-        dao.last_proposal_slot = last_proposal_slot;
+        dao.base_burn_lamports = base_burn_lamports;
 
         Ok(())
     }
 
-    pub fn set_base_burn_lamports(ctx: Context<Auth>, base_burn_lamports: u64) -> Result<()> {
+    pub fn set_burn_decay_per_slot_lamports(
+        ctx: Context<Auth>,
+        burn_decay_per_slot_lamports: u64,
+    ) -> Result<()> {
         let dao = &mut ctx.accounts.dao;
 
-        dao.base_burn_lamports = base_burn_lamports;
+        dao.burn_decay_per_slot_lamports = burn_decay_per_slot_lamports;
+
+        Ok(())
+    }
+
+    pub fn set_slots_per_proposal(ctx: Context<Auth>, slots_per_proposal: u64) -> Result<()> {
+        let dao = &mut ctx.accounts.dao;
+
+        dao.slots_per_proposal = slots_per_proposal;
+
+        Ok(())
+    }
```

Comment:
it might make more sense to include a single update params function with using `Option`s. that way you could update in conjunction with others more easily

example this: https://github.com/drift-labs/drift-competitions/blob/890830d0e76846f20fcf12036dd495b297bcd032/programs/drift-competitions/src/instructions/update_competition.rs#L51